content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -231,9 +341,3 @@ for x, y in list:
 Note: For "Why Did the Cow Cross the Road III", you need a custom comparator for Java. This is covered in [Silver](/silver/sorting-custom).
```

Comment:
> Note: For "Why Did the Cow Cross the Road III", you need a custom comparator for Java. 

probably don't need this line since custom comparator is mentioned just above?